hdk-rust/src/api.rs

```diff
@@ -896,6 +898,68 @@ pub fn close_bundle(_action: BundleOnClose) -> ZomeApiResult<()> {
     Err(ZomeApiError::FunctionNotImplemented)
 }
 
+/// Identical signature to get_links but also returns the entries that the links point to. The response is a 
+/// `GetLinksLoadResult<Entry>` which contains an `address` and `entry` field.
+/// This function does not care what type the entries are and can return a collection of 
+/// entries with different types
+pub fn get_links_and_load<S: Into<String>>(
+    base: &HashString,
+    tag: S
+) -> ZomeApiResult<GetLinksLoadResult<Entry>>  {
+    let get_links_result = get_links(base, tag)?;
+
+    Ok(get_links_result.addresses()
+    .iter()
+    .map(|address| {
+        get_entry(address.to_owned())
+        .map(|entry: Option<Entry>| {
+            GetLinksLoadElement{
+                address: address.to_owned(),
+                entry: entry.unwrap()
+            }
+        })
+    })
+    .filter_map(Result::ok)
```

Comment:
I think we should record that some of the get entries error out. Either by bubbling up the errors or returning a vec of results.

Reply:
Yes! I think that would be awesome

Reply:
I think this is now addressed.